modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -103,8 +103,21 @@ label:new[]
 DROP [COMPOSITE] DATABASE ... [RESTRICT \| CASCADE ALIAS[ES]]
 ----
 
-| Added the ability to drop any database aliases, that would otherwise block the deletion of the database, at the same time as the deletion of the database.
+| Added the ability to drop database aliases while deleting a database.
 This will affect local database aliases targeting the database and constituent database aliases belonging to the composite database.
+For more information, see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/standard-databases/delete-databases/#delete-databases-with-aliases[Delete a database with local database aliases targeting it] and link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/composite-databases/delete-composite-databases/#composite-databases-delete-with-aliases[Delete a composite database with constituent database aliases].
+
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+LOAD CSV FROM 'azb://azb-account/azb-container/artists.csv' AS row
+MERGE (a:Artist {name: row[1], year: toInteger(row[2])})
+RETURN a.name, a.year
+----
+
+| Extension of the xref:clauses/load-csv.adoc#azure-cloud-storage[LOAD CSV] clause to allow loading CSV files from Azure Cloud Storage URIs.
```

Comment:
feels weird that this is added as part of this PR...

Reply:
It wasnt - it was added here https://github.com/neo4j/docs-cypher/pull/1008
It appeared in this PR because I had to resolve a conflict

Reply:
because you pull-merged instead of rebased XD